src/clr-core/styles/tokens/tokens.yml

```diff
@@ -158,26 +158,84 @@ props:
     value: 200
     type: 'number'
     category: 'typography'
+    private: true
   - name: font-weight-regular
     value: 400
     type: 'number'
     category: 'typography'
+    private: true
```

Comment:
We did we make the font-weight typography helpers private?

Reply:
The attr utils no so I think this is a mistake on my part